src/cpdb/cpdb.py

```diff
@@ -22,10 +27,23 @@
 
 def cpdb():
     st.title("Capital Projects Database QAQC")
-    branch = st.sidebar.selectbox("select a branch", ["main"])
+    default_branch = get_default_branch(REPO_NAME)
+    branches = get_active_s3_folders(
+        repo=REPO_NAME, bucket_name=BUCKET_NAME
+    )
+    branch = st.sidebar.selectbox(
+        "Select a branch (will use latest)",
+        branches,
+        index=branches.index(default_branch),
+    )
```

Comment:
We/I are getting a little non-DRY here - defintely getting slightly repetitive with these selectboxes. Ideally we move to a file in `src/components.py` that takes in label, repo name, bucket name, etc but figure maybe that can wait for now

Reply:
Actually going to do a quick pass on this now